assets/content/cookbook/Design-patterns/singleton.md

```diff
@@ -16,6 +16,17 @@ class MySingleton {
 }
 ```
 
+More elegant way.
```

Comment:
I agree.. Maybe this should just replace the other example? For @markknol to decide.